tracer/src/Datadog.Trace/LibDatadog/TraceExporterConfiguration.cs

```diff
@@ -0,0 +1,162 @@
+// <copyright file="TraceExporterConfiguration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Datadog.Trace.LibDatadog;
+
+/// <summary>
+/// Represents a configuration for the trace exporter.
+/// </summary>
+internal class TraceExporterConfiguration : SafeHandle
```

Comment:
same here for the native calls
since we are calling into native, shouldn't we enclose with try/catch ?

Reply:
I think this one is indirectly wrapped in a try-catch as it's only used in `TracerManagerFactory` but @ganeshnj can confirm

Reply:
that's correct, we cover the init phase, which gives us better coverage instead of try-catch every single native call. Once the init passes (library was loaded successfully), there is no reason to do a try-catch.

Reply:
resolved in #7061